### DISCUSSION TITLE
Valley Bottom Geomorphic Unit Ratios

### DISCUSSION THREAD

**Discussion by lauren-herbine** [Riverscapes/riverscapes-tools#562] [General] [answered]
What metric do we want for GNAT and Mississippi reach typing?
What units?
Input data sources

**Comment by joewheaton** [comment]
> What metric do we want for GNAT and Mississippi reach typing?

We want active channel, active floodplain and inactive floodplain.

>What units?

These are fundamentally just proportion of valley bottom area, derived by taking there polygon areas (within the valley bottom sausage segment) and dividing by total valley bottom area. 
Intermediates it is helpful to have these in acres, square kilometers, square miles, square meters. 

>Input data sources

- Ocular estimate of proportions.
- Actual measurement of area from polygon areas (intersected to valley bottom sausage segment).

**Comment by joewheaton** [comment]
@lauren-herbine this issue sort of went dormant, but is essentially now part of VBET. 

@shelbysawyer we are displaying in "Estimated Valley Bottom" Geomorphic Units Active as likelihood 0.99 to 0.8 and inactive 0.8 to 0.68. From what I've seen, this is not a good estimate.  I am speculating that 0.9 will be better as a rough-cut of active vs. inactive. As such I have requested @KellyMWhitehead to change this in #415.

![image](https://user-images.githubusercontent.com/9723905/132144195-544fd489-2ee8-4c2e-a7e9-92390858293f.png)

![image](https://user-images.githubusercontent.com/9723905/132144261-3396a08b-8e65-4214-9450-a5f0bc588d3e.png)

**Comment by joewheaton** [comment]
This is answered.

**Comment by lauren-herbine** [comment, marked as answer]
## Active Channel to Valley Bottom Ratio

|               Attribute               |      Tool?      |         Line(s) of evidence needed          |           Measurements needed from LOE           |                         Calculation                          | Alpha (ready today) | Beta v 0.1 | V 1 (i.e. thesis) | V2 (i.e. later) |
| :-----------------------------------: | :-------------: | :-----------------------------------------: | :----------------------------------------------: | :----------------------------------------------------------: | :-----------------: | :--------: | :---------------: | :-------------: |
| Active channel to valley bottom ratio | GUT or **VBET** | Valley bottom segment + ChannelArea polygon | areas of channel area and valley bottom polygons | (𝑎𝑟𝑒𝑎 𝑜𝑓 𝑎𝑐𝑡𝑖𝑣𝑒 𝑐ℎ𝑎𝑛𝑛𝑒𝑙 𝑤𝑖𝑡ℎ𝑖𝑛 𝑣𝑎𝑙𝑙𝑒𝑦 𝑏𝑜𝑡𝑡𝑜𝑚 𝑠𝑒𝑔𝑚𝑒𝑛𝑡)/((𝑎𝑟𝑒𝑎 𝑜𝑓 𝑎𝑐𝑡𝑖𝑣𝑒 𝑓𝑙𝑜𝑜𝑑𝑝𝑙𝑎𝑖𝑛+𝑎𝑟𝑒𝑎 𝑜𝑓 𝑎𝑐𝑡𝑖𝑣𝑒 𝑐ℎ𝑎𝑛𝑛𝑒𝑙 + 𝑎𝑟𝑒𝑎 𝑜𝑓 *in*𝑎𝑐𝑡𝑖𝑣𝑒 𝑓𝑙𝑜𝑜𝑑𝑝𝑙𝑎𝑖𝑛)) |                     |            |       TRUE        |      FALSE      |

![vb_ratio](https://user-images.githubusercontent.com/73319684/163035466-884abcae-b7a4-4b29-80cd-50237ca190d2.png)



These areas are already calculated in VBET projects. I found them in the attribute tables for the `Valley Bottom Geomorphic Units`:

![image](https://user-images.githubusercontent.com/73319684/163037038-8c7f472d-94b2-4a8a-91bd-a6d86be31ade.png)

It looks like we are already calculating this metric in `prop_valley_bottom_pc`, though I am not sure what the "pc" is referring to. 

(𝑎𝑟𝑒𝑎 𝑜𝑓 𝑎𝑐𝑡𝑖𝑣𝑒 𝑐ℎ𝑎𝑛𝑛𝑒𝑙 𝑤𝑖𝑡ℎ𝑖𝑛 𝑣𝑎𝑙𝑙𝑒𝑦 𝑏𝑜𝑡𝑡𝑜𝑚 𝑠𝑒𝑔𝑚𝑒𝑛𝑡) / ((𝑎𝑟𝑒𝑎 𝑜𝑓 𝑎𝑐𝑡𝑖𝑣𝑒 𝑓𝑙𝑜𝑜𝑑𝑝𝑙𝑎𝑖𝑛 + 𝑎𝑟𝑒𝑎 𝑜𝑓 𝑎𝑐𝑡𝑖𝑣𝑒 𝑐ℎ𝑎𝑛𝑛𝑒𝑙 + 𝑎𝑟𝑒𝑎 𝑜𝑓 *in*𝑎𝑐𝑡𝑖𝑣𝑒 𝑓𝑙𝑜𝑜𝑑𝑝𝑙𝑎𝑖𝑛))

**Reply by lauren-herbine** [reply]
@joewheaton @philipbaileynar please let me know if you need more info/examples/etc!

**Comment by philipbaileynar** [comment]
`pc` just means percent. Analgous to `ha` for hectares.

**Reply by lauren-herbine** [reply]
Sweet, thank you!
